aspnetcore/includes/make-x509-cert.md

```diff
@@ -0,0 +1,7 @@
+To create self-signed X.509 certificates on Windows, options include:
+
+* [New-SelfSignedCertificate PowerShell cmdlet](/powershell/module/pkiclient/new-selfsignedcertificate?view=win10-ps)
+* [SelfCert](https://www.pluralsight.com/blog/software-development/selfcert-create-a-self-signed-certificate-interactively-gui-or-programmatically-in-net)
+* [OpenSSL](https://www.openssl.org/)
+
+On macOS and Linux, self-signed certificates can be created using [OpenSSL](https://www.openssl.org/).
```

Comment:
You can probably remove the link on the 2nd occurrence of OpenSSL. You already linked to it in the list above.

Reply:
True, but it reads "on Windows" and then "On mac and Linux." I'm concerned the Mac/Linux reader will just flat out *skip* that whole first bit and jump drop down to the "On macOS and Linux" bit.

Let me try a different setup of this content on the next commit that might fix it so that there's only one link to OpenSSL.